README.md

```diff
@@ -786,12 +786,12 @@ Developers in the project may not follow consistent code security practices, lea
 
 ## ![✔] 6.13. Run Node as non-root user
 
-**TL;DR:** Running on behalf of the root or full-privilege account means that any attacker can gain full control to perform anything within the machine if your application is compromised.
+**TL;DR:** In Docker environment, Nodejs runs as a root user by default with unlimited permission set. Create a noon-root user during image creation and run the container with the flag "-u username"
```

Comment:
@i0natan I'm not sure if I'd specifically refer to Docker here since many people just run the Node.js process on a barebones machine (i.e: no virtualization).

With that said, you're more detailed explanation is pretty good and provides a general description of the problem so either leave as is or just rephrase a bit.

Reply:
@lirantal I'm splitted here since 95% who runs node in bare metal doesn't run them as root, while many people use Docker and 95% of them run node as root...

Reply:
Like in my original comment, I'm also split between mentioning or not :-)
Thing is, you are being very specific. For example, what if technology changes tomorrow and docker is gone in favor of kube or something else? That's why I don't like this buzz word name dropping thing. Maybe replace "docker" with "container technology" or just "virtualization".